tmt/steps/execute/internal.py

```diff
@@ -559,7 +559,9 @@ def _run_tests(
                     else:
                         for result in invocation.results:
                             result.result = ResultOutcome.ERROR
-                            result.note = 'crashed too many times'
+                            result.note = """
+                            crashed too many times,you may want to set restart_max_count larger.
+                            """
```

Comment:
1. Multiline string will most likely carry those empty lines into the note. It might be better to split it into multiple strings:

```python
result.note = ('crashed too many times,'
               'you may want to set restart_max_count larger')
```

2. there is no `restart_max_count` object in user-visible part of tmt, the correct form is `restart-max-count`.
3. space after comma is needed, "times, you ..."